day_two_guide/topics/etcd_backup.adoc

```diff
@@ -0,0 +1,1167 @@
+////
+etcd tasks
+
+Module included in the following assemblies:
+
+* day_two_guide/host_level_tasks.adoc
+////
+
+etcd is the key value store for all object definitions, as well as the
+persistent master state. Other components watch for changes, then bring
+themselves into the desired state.
+
+{product-title} versions prior to 3.5 used etcd version 2, while 3.5 and later
+use v3. The data model between the two versions of etcd is different. etcd v3
+can use both the v2 and v3 data model, whereas etcd v2 can only use the v2 data
+model. In an etcd v3 server, the v2 and v3 data stores exist in parallel and
+are independent.
+
+For both v2 and v3 operations, you can use the `ETCDCTL_API` environment
+variable to use the proper API:
+
+----
+$ etcdctl -v
+etcdctl version: 3.2.5
+API version: 2
+$ ETCDCTL_API=3 etcdctl version
+etcdctl version: 3.2.5
+API version: 3.2
+----
+
+See xref:../install_config/upgrading/migrating_etcd.adoc[Migrating etcd Data (v2
+to v3) section] for information about how to migrate to v3.
+
+=== Backup and restore etcd
+
+The etcd backup process is composed of two different procedures:
+
+* Configuration backup: Including the required etcd configuration and
+certificates
+* Data backup: Including both v2 and v3 data model.
+
+The data backup procedure can be done on any host that has connectivity to the
+etcd cluster, where the proper certificates are provided, and where the
+`etcdctl` tool is installed.
+
+[NOTE]
+====
+The backup files must be copied to an external system other than the etcd
+members themselves, ideally outside the {product-title} environment, and those
```

Comment:
'and those files encrypted' confusing - what files? consider making another sentence or rewording.